sessions/js-structure/quiz-app/global.css

```
@@ -0,0 +1,66 @@
+:root {
+	--primary: #ffffff;
```

```suggestion
	--primary: #ffffff;
```
Why are `--primary` and `--white` the same color? Afaik `--primary` would be your primary and/or key color that your palette is based around.

---

Good point, I deleted the `--white` variable and replaced all instances with `--primary`.